### PR TITLE
Create PrintPrimeNumbersBetweenTwoNumbers.ts

### LINKED ISSUE #1306: Write a TypeScript program to print all prime numbers between two numbers
### Description

Write a TypeScript program to print all prime numbers between two numbers

```
Input  : 10 20
Output : 11 13 17 19
```
<details>
<summary>How to contribute</summary>

- Save the solution in `PrintAllPrimeNumbersBetweenTwoNumbers.ts` file
- Add `PrintAllPrimeNumbersBetweenTwoNumbers.ts` file in `print-all-prime-numbers-between-two-numbers` folder

</details>

### PR DESCRIPTION
**Pull request by greeshma-d**
Closes codinasion/codinasion#1038 

### What's being changed:

Added TS file to print prime numbers between 2 numbers

### CODE REVIEW

**Comment by greeshma-d** [comment]
True, will check and commit new changes

**Review by harshraj8843 — DISMISSED** [review]
Thanks for contributing 👍🏻

File: program/print-all-prime-numbers-between-two-numbers/PrintPrimeNumbersBetweenTwoNumbers.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function printPrimeNumber(num1: number, num2: number) {
+    let primeNumbers = 0;
```
Comment (victoriacheng15) [inline program/print-all-prime-numbers-between-two-numbers/PrintPrimeNumbersBetweenTwoNumbers.ts:2]: This line doesn't do anything. Maybe you could remove?

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Pull request merged**